src/app/src/components/Submissions/ActivityLog.js

```diff
@@ -1,31 +1,35 @@
 import { Fragment } from 'react';
 import { Text } from '@chakra-ui/react';
 
-export default function ActivityLog({ submissionId }) {
-    const fetchLogEntries = () => {
-        return [
-            [
-                'July 7, 2022 | 3:30 pm',
-                'Jenny Smith has submitted a new map for review',
-            ],
-        ];
-    };
+import { formatDateTime } from '../../utils';
 
-    const logEntries = fetchLogEntries();
-
-    const logEntriesAsTextElements = logEntries.map((entry, index) => {
-        const [timestamp, description] = entry;
-        return (
-            <Fragment key={index}>
-                <Text textStyle='submissionDetailHeading'>{timestamp}</Text>
-                <Text mt={2}>{description}</Text>
-            </Fragment>
-        );
-    });
+export default function ActivityLog({ entries }) {
+    const logEntriesAsTextElements = entries.map(
+        ({ user, action, time, notes }, index) => {
+            return (
+                <Fragment key={index}>
+                    <Text textStyle='submissionDetailHeading' mt={4}>
+                        {formatDateTime(time)}
```

Comment:
This should match this formatting:

```
July 7, 2022 | 3:30 pm
```

Reply:
Do we want to install an `npm` package for this?

Reply:
Let's use this for now:

```js
export function formatDateTime(date) {
    const d = new Date(date);
    const datePart = d.toLocaleDateString('en-US', { dateStyle: 'long' });
    const timePart = d.toLocaleTimeString('en-US', { timeStyle: 'short' });

    return `${datePart} | ${timePart}`;
}
```

```
October 24, 2022 | 9:28 AM
```